documentation/Maintenance.md

```diff
@@ -1835,6 +1836,48 @@ data:
 
 11. Check that everything works properly and remove the old ippool if necessary.
 
+## Kubelet server certificate approval
+
+The `kubelet` server certificate is self-signed by default. To avoid that one should change `cluster.yaml` in the following way:
```

Comment:
Wouldn't it be more clear for users to give here the path to the certificate (/var/lib/kubelet/pki/kubelet.crt)?